core/client/app/templates/setup/three.hbs

```diff
@@ -12,7 +12,8 @@
 sally.sanders@example.com" value=users}}
     </form>
 
-    <button {{action "signin"}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
+    <button {{action 'invite'}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
         {{buttonText}}
     </button>
+    <a {{action 'signin'}} href="#" class="gh-flow-skip">I'll do this later, take me to my blog!</a> 
```

Comment:
it's not hard - I can go back to the old spec. My thought with doing it this way is now, if the user quickly clicks the main CTA on the page, they won't automatically be taken off the invite page... That _should_ turn in to more invites, because the user has to make a more conscious decision to skip.

Also, having form status reflected in the button is nice.

But, if y'all prefer the old way, I'll go back.